.github/workflows/build_and_release.yml

```
@@ -0,0 +1,80 @@
+          INPUT_GITHUB_USERNAME: "yulin17-chang"
+          INPUT_GITHUB_TOKEN: ${{ secrets.TOKEN }}
+          INPUT_TARGET_REPOS: '[yulin17-chang/activity-log]'
+          INPUT_OUTPUT_STYLE: MARKDOWN
+          INPUT_IGNORE_EVENTS: '[]'
+          INPUT_README_PATH: README.md
+          INPUT_COMMIT_MESSAGE: Update README.md
+          INPUT_EVENT_LIMIT: '100'
+        run: node ./dist/index.js
+
+  release:
+    name : Release
```

yamllint

[warning] too many spaces before colon

(colons)